Pikipick/Pikipick/LocalNetworkPrivacy.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  LocalNetworkPrivacy.swift
+//  Pikipick
+//
+//  Created by woo0 on 2022/07/20.
+//
+
+import Foundation
+import Network
+import UIKit
+
+class LocalNetworkPrivacy : NSObject {
+	let service: NetService
+
+	var completion: ((Bool) -> Void)?
+	var timer: Timer?
+	var publishing = false
```

Comment:
Bool 값은 의문형으로 네이밍된다면 알기 쉽답니다

```suggestion
	let service: NetService

	var completion: ((Bool) -> Void)?
	var timer: Timer?
	var isPublishing = false
```

Reply:
넵 미처 신경쓰지 못한 거 같네요ㅠ 참고하겠습니다!